web/src/app/workplace/services/task.service.ts

```diff
@@ -102,4 +110,24 @@ export class TaskService {
   createTask(task: Task): Observable<Task> {
     return this.httpClient.post<Task>(this.url, task);
   }
+
+  private convertDatesWithTimeZoneBeforeToShow(task: Task): Task {
```

Comment:
I prefer applyTaskDatesTimeZone() and instead of TaskanaDate.transformDateWithTimeZone(task.modified) i  would prefer TaskanaDate.applyTimeZone(task.modified)